### ISSUE TITLE
HANA Functional Correctness

### ISSUE THREAD

**Issue by ghost** [abap2xlsx/abap2xlsx#413]
Hello Ivan and Stefan,

in the context of the delivery of abap2xlsx our architecture team found an error of HANA functional correctness in the class ZCL_EXCEL_WORKSHEET in the method CALCULATE_CELL_WIDTH in line 153:

```
SELECT tdfamily
  FROM tfo01
  INTO TABLE lt_font_families
  UP TO 1 ROWS
  WHERE tdtext = ld_font_name.
```

should be replaced by

```
SELECT tdfamily
  FROM tfo01
  INTO TABLE lt_font_families
  UP TO 1 ROWS
  WHERE tdtext = ld_font_name
  ORDER BY PRIMARY KEY.
```

Is it possible for you to change it in the next release of abap2xlsx?

Thanks in advance.

Cheers
Stefan

**Comment by gregorwolf** [comment]
Dear Stefan,

would be the best if you fork, correct and then send a pull request.

Best regards
Gregor

**Comment by ghost** [comment]
Hello Gregor,

thank you very much for the tip - it is the first time for me to fork a repository. Hope I done well.

Cheers
Stefan

**Comment by gregorwolf** [comment]
I've merged your pull request #414
